ros2_batch_job/__main__.py

```diff
@@ -373,11 +361,6 @@ def run(args, build_function, blacklisted_package_names=None):
     args.buildspace = 'build space' if 'buildspace' in args.white_space_in else 'build'
     args.installspace = 'install space' if 'installspace' in args.white_space_in else 'install'
 
-    if args.disable_connext_static:
-        os.environ["CONNEXT_STATIC_DISABLE"] = '1'
-    if args.disable_connext_dynamic:
-        os.environ["CONNEXT_DYNAMIC_DISABLE"] = '1'
```

Comment:
This custom logic is not necessary anymore with each RMW impl. being ignored by its name.